source/_posts/2023-04-27-year-of-the-voice-chapter-2.markdown

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "Year of the Voice - Chapter 2: Let's talk"
+description: "Talk to your smart home and let it talk back with our new voice assistant features."
+date: 2023-04-27 00:00:00
+date_formatted: "April 27, 2023"
+author: Paulus Schoutsen
+comments: true
+categories: Assist
+og_image: /images/blog/2023-04-27-year-of-the-voice-chapter-2/social.png
+---
+
+
+This year is Home Assistant’s [Year of the Voice]. It is our goal for 2023 to let users control Home Assistant in their own language. Today we’re presenting Chapter 2, our second milestone building towards this goal.
+
+In [Chapter 1] we focused on intents – what the user wants to do. As of today, the Home Assistant community has translated common smart home commands and responses into [45 languages], closing in on the 62 languages supported by Home Assistant.
```

Comment:
```suggestion
In [Chapter 1], we focused on intents – what the user wants to do. Today, the Home Assistant community has translated common smart home commands and responses into [45 languages], closing in on the 62 languages that Home Assistant supports.
```

Reply:
Should we say "currently supported by"?

Reply:
The blog is already in the context of Home Assistant, so I guess that is not needed.